src/inmanta/agent/agent.py

```diff
@@ -667,8 +683,10 @@ def periodic_schedule(
                 )
             )
             self.ensure_deploy_on_start = False
-        periodic_schedule("deploy", deploy_action, self._deploy_interval, self._deploy_splay_value, now)
-        periodic_schedule("repair", repair_action, self._repair_interval, self._repair_splay_value, now)
+
+        periodic_schedule("deploy", deploy_action, self._deploy_interval, self._deploy_splay_value)
+        periodic_schedule("repair", repair_action, self._repair_interval, self._repair_splay_value)
+        periodic_schedule("executor cleanup", cleanup_executors_action, self._executor_retention, 0)
```

Comment:
I don't think this produces the expected behavior: it clean up every  `executor-retention` it doesn't clean after `executor-retention` 


(i.e. clean up after 24h is not the same as cleaning up every 24h)